src/test/java/core/basesyntax/StartProcessTest.java

```diff
@@ -0,0 +1,171 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.shop.item.Fruit;
+import core.basesyntax.shop.service.FileReader;
+import core.basesyntax.shop.service.FileReaderImpl;
+import core.basesyntax.shop.service.OperationsStrategy;
+import core.basesyntax.shop.service.OperationsStrategyImpl;
+import core.basesyntax.shop.service.ShopService;
+import core.basesyntax.shop.service.ShopServiceImpl;
+import core.basesyntax.shop.service.operations.Balance;
+import core.basesyntax.shop.service.operations.OperationHandler;
+import core.basesyntax.shop.service.operations.Purchase;
+import core.basesyntax.shop.service.operations.Return;
+import core.basesyntax.shop.service.operations.Supply;
+import core.basesyntax.validators.ValidatorForFile;
+import core.basesyntax.validators.ValidatorForFileImpl;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class StartProcessTest {
+
+    private static final Fruit BANANA0 = new Fruit("banana", 0);
+    private static final Fruit BANANA30 = new Fruit("banana", 30);
+    private static final String[] FRUITS = new String[]{"type,fruit,quantity",
+            "s,banana,20"};
+
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+
+    private Map<String, OperationHandler> operationMap = new HashMap<>();
+
+    private OperationsStrategy operationsStrategy = new OperationsStrategyImpl(operationMap);
+    private ShopService shop = new ShopServiceImpl(new FruitDaoImpl(), operationsStrategy);
+
+    @After
+    public void init() {
```

Comment:
```suggestion
    public void tearDown() {
```
naming `init` make sense when you use `Before` annotation

Reply:
+